src/Middleware/Diagnostics.EntityFrameworkCore/src/DiagnosticsEntityFrameworkCoreLoggerExtensions.cs

```diff
@@ -75,6 +75,9 @@ internal static class DiagnosticsEntityFrameworkCoreLoggerExtensions
             new EventId(7, "DatabaseErrorPageException"),
             "An exception occurred while calculating the database error page content. Skipping display of the database error page.");
 
+        private static readonly Action<ILogger, Exception?> _responseStartedDatabaseDeveloperPageExceptionFilter =
+            LoggerMessage.Define(LogLevel.Warning, new EventId(1, "ResponseStarted"), "The response has already started, the database developer page exception filter will not be executed.");
```

Comment:
The DB filter already failed, it's the next filter that won't be executed, right?